velero-csi-plugin/csi_snapshotter.go

```diff
@@ -38,30 +38,27 @@ import (
 	"github.com/vmware-tanzu/velero/pkg/util/boolptr"
 )
 
-const (
-	veleroSnapshotNameAnnotation = "velero.io/volume-snapshot-name"
-)
-
 // CSISnapshotter is a backup item action plugin for Velero.
 type CSISnapshotter struct {
 	log logrus.FieldLogger
 }
 
-// AppliesTo returns information about which resources this action should be invoked for.
-// A BackupPlugin's Execute function will only be invoked on items that match the returned
+// AppliesTo returns information about which resources the CSISnapshotter action should be invoked for.
+// CSISnapshotter BackupItemAction plugin's Execute function will only be invoked on items that match the returned
 // selector. A zero-valued ResourceSelector matches all resources.
 func (p *CSISnapshotter) AppliesTo() (velero.ResourceSelector, error) {
-	p.log.Info("CSISnapshotter AppliesTo")
+	p.log.Info("CSISnapshotterAction AppliesTo")
 
 	return velero.ResourceSelector{
 		IncludedResources: []string{"persistentvolumeclaims"},
 	}, nil
 }
 
-// Execute allows the ItemAction to perform arbitrary logic with the item being backed up,
-// in this case, setting a custom annotation on the item being backed up.
+// Execute allows the RestorePlugin to perform arbitrary logic with the item being restored,
```

Comment:
I would drop "allows the RestorePlugin to perform arbitrary logic" bit and just document *this* plugin's behavior.